.github/workflows/cypress.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         run: |
           export API_KEY=`sudo maas apikey --username=admin`
           maas login admin http://localhost:5240/MAAS $API_KEY
+          while [ $(maas admin boot-resources is-importing | cat) == "true" ]; do sleep 10; done; echo "syncing finished"
```

Comment:
I think ideally it should be moved to a separate step (e.g. with a name "Wait for MAAS boot resources") which would help track its progress (and execution time).